demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -205,6 +204,11 @@ export enum ContentShareType {
   VideoFile,
 }
 
+enum StreamDirection {
```

Comment:
Can we just reuse ClientMetricReportDirection? its weird that you create a new one to compare to a different enum.